test/es-module/test-typescript.mjs

```diff
@@ -324,3 +324,30 @@ test('execute a JavaScript file importing a cjs TypeScript file', async () => {
   match(result.stdout, /Hello, TypeScript!/);
   strictEqual(result.code, 0);
 });
+
+test('execute a TypeScript file with union types', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--experimental-strip-types',
+    '--no-warnings',
+    fixtures.path('typescript/ts/test-union-types.ts'),
+  ]);
+
+  strictEqual(result.stderr, '');
+  strictEqual(result.stdout,  `{ name: 'Hello, TypeScript!' }\n{ role: 'admin', permission: 'all' }\n{\n  foo: 'Testing Partial Type',\n  bar: 42,\n  zoo: true,\n  metadata: undefined\n}\n`)
```

Comment:
This line is too long, can you shorten it with string concatenation?

Reply:
Updated! 🙏